jenkins/plugins.txt

```diff
@@ -1,104 +1,103 @@
-swarm:3.15
-git:4.0.0-rc
-git-client:3.0.0-rc
-git-server:1.7
-mailer:1.23
-scm-api:2.3.0
-matrix-project:1.14
-ssh-credentials:1.14
-credentials:2.1.18
-script-security:1.54
-role-strategy:2.10
-matrix-auth:2.3
-icon-shim:2.0.3
-authentication-tokens:1.3
+swarm
```

Comment:
So the new strategy is to stop tracking pinned versions of the plugins? From my side that would reduce maintenance but I wonder what the impact will be when breaking changes are made to the plugin. Would this prevent a deployment as the job configuration are not in-sync?

Reply:
Pinning the version leads to errors again i.e. unable to find plugins etc. 
Test:
 * install without pinning
 * run script to get the version of each plugin
 * install with pin version
 * failure